Makefile

```diff
@@ -173,6 +174,9 @@ help:
 	@echo '                      sourced from ceph-container (e.g., ubuntu:"16.04", centos:"7")'
 	@echo '    e.g., FLAVORS="luminous,ubuntu,16.04 jewel,ubuntu,14.04"'
 	@echo ''
+	@echo '  REGISTRY - The url of the registry to pull images from.'
+	@echo '             Default to "_" which means the default docker registry'
```

Comment:
I think "docker" here could be problematic. Docker (the company) has a copyright on the word "docker" in this context. I would suggest just leaving it out to remove any possibility of Docker (the company) taking legal action.